x/mint/keeper/migrator.go

```diff
@@ -21,3 +21,9 @@ func NewMigrator(k Keeper) Migrator {
 func (m Migrator) Migrate1to2(ctx context.Context) error {
 	return nil
 }
+
+// Migrate2to3 migrates the x/mint module state from the consensus version 2 to
+// version 3.
+func (m Migrator) Migrate2to3(ctx context.Context) error {
+	return nil
```

Comment:
Why is ths nil? I think we should get the params and set MaxSupply with the default value (0)

Reply:
my bad, I thought I pushed this code already, will do now